lib/etl/create_new_resource_record.rb

```diff
@@ -0,0 +1,10 @@
+class CreateNewResourceRecord
+  def write(attributes)
+    record = Resource.create!(attributes)
+
+    ap record.metadata
```

Comment:
Yes, done in 523e706 and added a little more text so it's clear what's going on.